Testscripts/Linux/validate-da.sh

```diff
@@ -56,6 +55,10 @@ function fail_test() {
     exit 0
 }
 
+function sleep_log_message() {
+    LogMsg "Sleeping for $1s"
```

Comment:
Log the message and do the sleep in the function.

For readability `"... for $1 seconds"`. These are for human readers. Over use of abbreviations can cause people to have to think more about the text of the message before they can absorb the meaning of the message.

Reply:
Done